packages/app/src/sandbox/eval/transpilers/base64/mimes.json

```diff
@@ -30,6 +30,8 @@
   "fst": "image/vnd.fst",
   "mmr": "image/vnd.fujixerox.edmics-mmr",
   "rlc": "image/vnd.fujixerox.edmics-rlc",
+  "txt": "text/plain",
+  "md": "text/plain",
```

Comment:
This could also be `text/markdown` but not sure if that's still in draft or already the considered standard for markdown